### ISSUE TITLE
key_states() always return [False, False, ..., False] on Windows but registers keys fine on Linux

### ISSUE THREAD

**Issue by Giraut** [abcminiuser/python-elgato-streamdeck#135]
This works fine in Linux:

```
from StreamDeck.DeviceManager import DeviceManager

streamdeck = DeviceManager().enumerate()[0]
streamdeck.open()

while not any(streamdeck.key_states()):
  pass

print("Key pressed")

streamdeck.close()
```

It never quits in Windows: key_states() never registers any keys and always returns a list of False states.

This is just a test code to highlight the issue, but I have a full application that uploads images to the keys, and it works on Windows as well as Linux. It's only key_states() that doesn't register keys.

Any idea?

Windows 11
Stream Deck Mk2
Python version 3.10.2
streamdeck version 0.9.5
hidapi.dll version 0.14.0

Thanks!

**Comment by abcminiuser** [comment]
This works on my Windows machine, however note that the current 0.9.5 version of the library requires a callback set for the key states to be updated (without a callback set the reader thread will not update the cached key states).

I'm using the stock Python interpreter. It's not a good idea to spinloop like in your example code however - sleeping for a few milliseconds will greatly reduce the CPU usage.

**Comment by abcminiuser** [comment]
Polling of keys with no callback set is now implemented in 9cbcc113920bda159f1792b886f4a3e98173fd12 .
